src/scss/_colors.scss

```diff
@@ -0,0 +1 @@
+$white: #fff;
```

Comment:
Define all the colours on a map 
https://sass-lang.com/documentation/variables
refer to the last section of this.
why is only a single colour imported here?